src/Scrabble/Search.hs

```diff
@@ -54,6 +56,22 @@ containsAll :: String -> Search
 containsAll s1 s2 = fst $ foldl' f (True, ups s2) (ups s1) where
   f (b,s) c = if elem c s then (b, delete c s) else (False,s)
 
+-- | Like the above, but if it encounters a character in in s1 not in
+--   s2, it deletes a blank and tries again before returning false
+containsAllWithBlanks :: String -> Search
```

Comment:
Sure. But what do you think about this idea. It's a bit harder, but, maybe worth it. Making a new module `Scrabble.ScrabbleSearch`, which imports and exports `Scrabble.Search`. Move can import `Scrabble.ScrabbleSearch`. The reason for this, you ask? Because I want to cheat on the command line :) And so these functions are actually useful for that. But, still keeping `Scrabble.Search` agnostic is worth keeping, because it's a nice little independent combinator library. Maybe we can rename `Search` to `SearchBase` so that it is a little more clear. Thoughts?